### ISSUE TITLE
Docker Images

### ISSUE THREAD

**Issue by LynHyper** [ipns-link/ipns-link#4]
A publicly available docker image, or instructions on how to build an ipns-link docker image, would be a nice addition in helping people integrate ipns-link into their server setups.

**Comment by SomajitDey** [comment]
@LynHyper Indeed. @ch4itu also requested this feature. Unfortunately, I don't yet know enough about how to dockerize Bash scripts. Can you or @rinzler100 help?

**Comment by rinzler100** [comment]
I could give it a shot but I'm not familiar with the process either...
